packages/devtools_app/lib/src/timeline/flutter/flutter_frames_chart.dart

```diff
@@ -149,22 +178,8 @@ class _FlutterFramesChartState extends State<FlutterFramesChart>
       // TODO(kenz): make this a general background color for use throughout
       // devtools.
       backgroundColor: chartBackgroundColor,
-      axisLeftSettingFunction: (axisLeft, controller) {
-        axisLeft
-          ..setStartAtZero(true)
-          ..typeface = lightTypeFace
-          ..textColor = defaultForeground
-          ..drawGridLines = false
-          ..setValueFormatter(YAxisUnitFormatter())
-          ..addLimitLine(LimitLine(60, '60 FPS')
-            // TODO(terry): LEFT_TOP is clipped need to fix in MPFlutterChart.
-            ..labelPosition = LimitLabelPosition.RIGHT_TOP
-            ..textSize = 10
-            ..typeface = boldTypeFace
-            // TODO(terry): Below crashed Flutter in Travis see issues/1338.
-            // ..enableDashedLine(5, 5, 0)
-            ..lineColor = const Color.fromARGB(0x80, 0xff, 0x44, 0x44));
-      },
+      // The axisLeftSettingFunction is computed in the frameAddedNotifier listener
+      // using the displayRefreshRate in FrameBasedTimeline.
```

Comment:
Update comment to match where function is computed if you can move it out to didChangeDeoendencies

Reply:
Done.